### DISCUSSION TITLE
How to print MetaTensor without metadata

### DISCUSSION THREAD

**Discussion by glemeur** [Project-MONAI/MONAI#7075] [Q&A]
Hello Monai community,

First of all, thank you very much for the development of this great tool!

I have a very basic question for which I could not find an answer in the doc: **how do print a (meta) tensor without tensor metadata and applied operation**? I've tried many things (like `MetaTensor.as_tensor()`) but the metadata is always appended to the output. 

Thanks a lot.

**Comment by wyli** [comment]
thanks! which version of MONAI are you using? the latest shouldn't have this issue
```py
>>> from monai.data import MetaTensor
>>> MetaTensor(1.0)
metatensor(1.)
>>> print(MetaTensor(1.0))
metatensor(1.)
>>> MetaTensor(1.0).print_verbose()
metatensor(1.)
{affine: tensor([[1., 0., 0., 0.],
        [0., 1., 0., 0.],
        [0., 0., 1., 0.],
        [0., 0., 0., 1.]], dtype=torch.float64), space: RAS}

```

or you can convert it into a regular tensor:
```
>>> from monai.data import MetaTensor
>>> import torch
>>> MetaTensor(1.0).as_subclass(torch.Tensor)
tensor(1.)
```
